topics/transcriptomics/tutorials/RNA_seq_analysis/tutorial.md

```diff
@@ -0,0 +1,542 @@
+- This course WILL give you the basis information to understand and run a generic RNA-seq analysis, its key steps and problematics, and how to interact with bioinformaticians/bioanalysts that can analyze your RNA-seq datasets
+contributions:
+  authorship:
+    - kamimrcht
+    - ppericard
+  editing:
+    - agathebq
+    - boucherblandine
+    - KhoudiaThiam
+---
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Introduction
+
+We chose D. melanogaster for its relatively small genome (to scale for the time/memory requirements of our experiments), for its ability to produce alternative isoforms (as a Eukaryota), and because it is a well-known species: a reference genome is available.
+This is where we got the data: https://www.ncbi.nlm.nih.gov/sra/SRX675217
+
+![ncbi D. melanogaster screenshot](./image/ncbi.jpg)
+
+ <question-title>Question</question-title>
+>
+> 1. Write here the characteristics of the complete sequencing dataset:
+>    
+>    ●	Run accession id
+>    
+>    ●	Sequencing technology
+>    
+>    ●	Library preparation
+>    
+>    ●	Single / Paired-end ?
+>    
+>    ●	Number of reads 
+>
+> > <solution-title>Solution</solution-title>
+> >
+> >     ●	Run accession id: SRX675217
+> >     ●	Sequencing technology: Illumina HiSeq 2000
+> >     ●	Library preparation: RNA-seq (cDNA)
+> >     ●	Single / Paired-end ?:  Single
+> >     ●	Number of reads: 10.4M
+> >
+> {: .solution}
+>
+{: .question}
+
+For this practical workshop, we built a reduced dataset (11421 SE reads):
+1.	9 regions of interest from the 2L chromosome were selected
+2.	Reads that aligned onto those regions were extracted (= 7245 reads)
+3.	Reads with adapter sequences were added (+ 2176 reads)
+4.	rRNA reads were added (+ 2000 reads)
+
+
+#  Cleaning - Pre-processing
+
+## Datasets preparation
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Check that the datatype
+{: .hands_on}
+
+## Initial quality control with **FastQC**
+
+> <hands-on-title> Run FastQC on the raw dataset </hands-on-title>
+>
+> {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.74+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Raw read data from your current history"*: `SRR1543719.sample.fastqsanger`
+{: .hands_on}
+
+> <question-title>Question</question-title>
+> 1. How many reads are in this dataset?
+> 2. What sequencing length was used?
+> 3. How would you qualify the sequencing quality of this run?
+> 4. Are there any quality problems we should deal with? 
+>
+> > <solution-title>Solution</solution-title>
```

Comment:
```suggestion
> <question-title></question-title>
> 1. How many reads are in this dataset?
> 2. What sequencing length was used?
> 3. How would you qualify the sequencing quality of this run?
> 4. Are there any quality problems we should deal with? 
>
> > <solution-title></solution-title>
```

the words 'question' and 'solution' inside aren't necessary, they will be included automatically by the template already.